docker.sh

```diff
@@ -2,7 +2,7 @@
 
 TASK_NAME=$1;
 
-export PROJECT_NAME=$(cat ./.project_name)
+export ENABLED_XDEBUG="OFF"
```

Comment:
Could you rename to `PROJECT_XDEBUG_ENABLED`?
Thanks to that we can list only env vars that describe a project

btw.
I'm not sure it's a good practice to control xdebug by `string` instead of `boolean`.